sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultJcaProvider.java

```diff
@@ -86,6 +86,46 @@ private void initialize() {
                     null
                 )
             );
+            putService(
+                new Service(
+                    this,
+                    "Signature",
+                    "RSASSA-PSS",
```

Comment:
About these strings: "RSASSA-PSS", "SHA256withECDSA", "SHA384withECDSA", "SHA512withECDSA".

1. Where did you get the string?
2. How about write them like `public static final String ` in some where?

Reply:
How about add 2 columns in the table in readme? One column is about `"RSASSA-PSS"`, another one is about `KeyVaultKeyLessRsaSignature`.

Reply:
I put them in the same column because they are essentially signature algorithms, not sure if this is appropriate